modules/osdk-about-scorecard-tool.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-scorecard.adoc
+
+[id="osdk-about-scorecard-tool_{context}"]
+= About the scorecard tool
+
+To validate an Operator, the Operator SDK's scorecard tool begins by creating
+all resources required by any related Custom Resources (CRs) and the Operator.
+The scorecard then creates another container in the Operator's Deployment which
+is used to record calls to the API server and run many of the tests. The tests
+performed also examine some of the parameters in the CRs.
+
+The scorecard also supports plug-ins, which allows extended functionality
```

Comment:
in v0.15 the scorecard quit supporting externally provided plug-ins, so that could be removed from this paragraph.  The scorecard would 'add' a proxy container into the Operator's deployment enabling the collection of API calls to the server, used in 2 of the internal tests.